libs/api/index.js

```diff
@@ -0,0 +1,13 @@
+// TODO: error catch
+export const queryTokenUsage = async () => {
+  const token = await AP.context.getToken();
+  const response = await fetch(`/client/info?jwt=${token}`, {
+    method: 'GET'
+  })
+
+  const result = await response.json()
+
+  if (result) {
```

Comment:
I saw your "TODO" comment above. For the sake of tech discussion, Line 10~12 has no real effect, right? It will return result anyway.
Btw, happy to merge without this TODO being done.

Reply:
Yes, there is a "TODO" remaining here. will improve it by abstracting the `request` function based on `fetch`, which will enable us to handle all types of data fetching situations with APIs.